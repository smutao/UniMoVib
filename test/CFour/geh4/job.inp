a test job

 $contrl
   qcprog="cfour"
 $end

 $qcdata
   fchk="geh4.out"
   geom="GRD"
 $end
