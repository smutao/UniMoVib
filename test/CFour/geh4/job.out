 =======================================================================================================================
       _/     _/                _/     _/            _/     _/
      _/     _/            _/  _/|  _/_/            _/     _/  _/  _/        UniMoVib:          A unified interface for
     _/     _/   _/_/_/       _/ |_/ _/   _/_/_/   _/     _/      _/           molecular harmonic vibrational frequency
    _/     _/  _/    _/  _/  _/  _/ _/  _/    _/  _/     _/  _/  _/_/_/        calculations.
   _/     _/  _/    _/  _/  _/     _/  _/    _/  _/    _/   _/  _/   _/
  _/     _/  _/    _/  _/  _/     _/  _/    _/   _/  _/    _/  _/   _/       Ver. 1.0.0, MAY 02, 2017
   _/_/_/   _/    _/  _/  _/     _/   _/_/_/     _/_/     _/  _/_/_/         https://github.com/zorkzou/UniMoVib
 =======================================================================================================================


 QC Program:      CFOUR
 FCHK file:       geh4.out
 GEOM file:       GRD


 Cartesian coordinates (Angstrom)
 ------------------------------------------------------------------------------------------
   No.   Atom    ZA                 X             Y             Z                Mass
 ------------------------------------------------------------------------------------------
     1    Ge     32            0.00000000    0.00000000    0.00000000           73.92117740
     2    H       1            0.00000000   -1.25868384    0.89002388            1.00782503
     3    H       1            0.00000000    1.25868384    0.89002388            1.00782503
     4    H       1            1.25868384    0.00000000   -0.89002388            1.00782503
     5    H       1           -1.25868384    0.00000000   -0.89002388            1.00782503
 ------------------------------------------------------------------------------------------


 <<< SYMMETRY >>>
 Molecular Point Group (Z)   :   Td  
 Molecular Point Group (Z+M) :   Td  


 ************************************
 ***  Properties of Normal Modes  ***
 ************************************

 Results of vibrations:
 Normal frequencies (cm**-1), reduced masses (AMU),
 force constants (mDyn/A), IR intensities (km/mol)

                                                   1                                 2                                 3
          Irreps                                  T2                                T2                                T2
     Frequencies                            835.0248                          835.1145                          835.3904
  Reduced masses                              1.0449                            1.0449                            1.0449
 Force constants                              0.4293                            0.4294                            0.4297
  IR intensities                            121.2451                          121.2941                          121.3032
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1  32         0.00000  -0.00000  -0.02256      -0.00000   0.02256  -0.00000      -0.02256  -0.00000  -0.00000
           2   1        -0.00000   0.28092   0.41390       0.00000  -0.21522  -0.28064       0.61210   0.00000   0.00000
           3   1        -0.00000  -0.28092   0.41390       0.00000  -0.21522   0.28064       0.61210   0.00000  -0.00000
           4   1         0.28053   0.00000   0.41330       0.00000  -0.61210   0.00000       0.21524   0.00000   0.28063
           5   1        -0.28053   0.00000   0.41330      -0.00000  -0.61210   0.00000       0.21524   0.00000  -0.28063

                                                   4                                 5                                 6
          Irreps                                   E                                 E                                T2
     Frequencies                            926.0930                          926.2148                         2160.9759
  Reduced masses                              1.0078                            1.0078                            1.0237
 Force constants                              0.5093                            0.5094                            2.8166
  IR intensities                              0.0001                            0.0000                          130.5883
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1  32         0.00000   0.00000   0.00002      -0.00000  -0.00000   0.00000       0.00000  -0.00000  -0.01476
           2   1        -0.00000   0.28852   0.40793       0.50000   0.00000   0.00000      -0.00000  -0.42196   0.27167
           3   1         0.00000  -0.28852   0.40793      -0.50000  -0.00000   0.00000      -0.00000   0.42196   0.27167
           4   1        -0.28893   0.00000  -0.40849      -0.00000  -0.50000  -0.00000      -0.41883  -0.00000   0.26949
           5   1         0.28893  -0.00000  -0.40849       0.00000   0.50000  -0.00000       0.41883  -0.00000   0.26949

                                                   7                                 8                                 9
          Irreps                                  T2                                T2                                A1
     Frequencies                           2160.9816                         2161.0472                         2167.2366
  Reduced masses                              1.0237                            1.0237                            1.0078
 Force constants                              2.8166                            2.8167                            2.7890
  IR intensities                            130.5374                          130.5270                            0.0019
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1  32        -0.00000  -0.01475   0.00000       0.01475   0.00000  -0.00000      -0.00000   0.00000   0.00006
           2   1         0.00000   0.56780  -0.42045       0.02683  -0.00000   0.00000       0.00000  -0.40664   0.28770
           3   1         0.00000   0.56780   0.42045       0.02683   0.00000  -0.00000       0.00000   0.40664   0.28770
           4   1         0.00000  -0.02681  -0.00000      -0.56779  -0.00000   0.42046       0.40978   0.00000  -0.28975
           5   1        -0.00000  -0.02681  -0.00000      -0.56779  -0.00000  -0.42046      -0.40978   0.00000  -0.28975

 Results of translations and rotations:

                                                   1                                 2                                 3
          Irreps                                  T2                                T2                                T2
     Frequencies                             -0.8234                           -1.0084                           -1.1644
  Reduced masses                             15.5905                           15.5905                           15.5905
 Force constants                             -0.0000                           -0.0000                           -0.0000
  IR intensities                              0.0000                            0.0000                            0.0000
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1  32         0.44721   0.00000   0.00000       0.00000   0.44721   0.00000       0.00000   0.00000   0.44721
           2   1         0.44721   0.00000   0.00000       0.00000   0.44721   0.00000       0.00000   0.00000   0.44721
           3   1         0.44721   0.00000   0.00000       0.00000   0.44721   0.00000       0.00000   0.00000   0.44721
           4   1         0.44721   0.00000   0.00000       0.00000   0.44721   0.00000       0.00000   0.00000   0.44721
           5   1         0.44721   0.00000   0.00000       0.00000   0.44721   0.00000       0.00000   0.00000   0.44721

                                                   4                                 5                                 6
          Irreps                                  T1                                T1                                T1
     Frequencies                             52.4789                           56.7890                           55.4361
  Reduced masses                              1.0078                            1.0078                            1.0078
 Force constants                              0.0016                            0.0019                            0.0018
  IR intensities                              0.0000                            0.0000                            0.0000
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1  32         0.00000  -0.00000   0.00000       0.00000   0.00000   0.00000       0.00000   0.00000   0.00000
           2   1         0.00000   0.35355   0.50000      -0.50000  -0.00000  -0.00000      -0.35355   0.00000   0.00000
           3   1         0.00000   0.35355  -0.50000       0.50000   0.00000   0.00000      -0.35355   0.00000   0.00000
           4   1         0.00000  -0.35355   0.00000       0.00000  -0.50000   0.00000       0.35355   0.00000   0.50000
           5   1         0.00000  -0.35355   0.00000       0.00000   0.50000   0.00000       0.35355   0.00000  -0.50000


 *********************************************
 ***   Thermal Contributions to Energies   ***
 *********************************************

 Temperature               :       298.150000    Kelvin
 Pressure                  :         1.000000    Atm
 Molecular mass            :        77.952478    AMU
 Elec. total energy        :         0.000000    Hartree
 Scale factor of Frequency :         1.000000
 Rot. symmetry number      :        12
 The Td   point group is used to calculate rotational entropy.

 Principal axes and moments of inertia in atomic units:
                                     1                   2                   3
     Eigenvalues --                 22.807439           22.807439           22.807439
           X                         1.000000            0.000000            0.000000
           Y                         0.000000            0.000000            1.000000
           Z                         0.000000            1.000000            0.000000

 Rotational temperatures             3.797612            3.797612            3.797612    Kelvin
 Rot. constants A, B, C              2.639476            2.639476            2.639476    cm**-1
                                    79.129497           79.129497           79.129497    GHz

 Thermal correction energies                              Hartree            kcal/mol
 Zero-point Energy                          :            0.029635           18.595979
 Thermal correction to Energy               :            0.032773           20.565127
 Thermal correction to Enthalpy             :            0.033717           21.157613
 Thermal correction to Gibbs Free Energy    :            0.009030            5.666455


 Job terminated correctly! Tue May 02 17:52:34 2017
